fastlane_bot/bot.py

```diff
@@ -354,12 +350,25 @@ def _convert_trade_instructions(
                 "ConfigObj": self.ConfigObj,
                 "db": self.db,
```

Comment:
Just in case you missed that part in my previous suggestion, you can add here:
```
"strategy_id": self.db.get_pool(cid=ti["cid"].split('-')[0]).strategy_id
```
Then get rid of function `_add_strategy_id_to_trade_instructions_dic`.
It would be overall cleaner IMO.
Feel free to resolve this comment regardless of your decision :)